### ISSUE TITLE
`block-opening-brace-newline-after` says nothing about a block holding nothing but comments

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#672]
### What minimal example or steps are needed to reproduce the bug?

`block-opening-brace-newline-after` measures the whitespace in front of the first non-comment node of a block, so that a comment standing at the head of the block may hold the break the option asks for. A block whose children are comments alone has no such node: the walk comes back with nothing and the check ends before anything is measured, so no option ever says a word about such a block, whatever stands behind the opening brace.

`always`, over a block holding no break behind the brace at all:

```css
a {/*c*/}
```

```css
a {/*1*/ /*2*/}
```

```css
@media print {/*c*/}
```

Not one of the three is reported. The same block with a declaration behind the comment is:

```css
a {/*c*/ color: pink; }
```

`1:3` — `Expected newline after "{"`. So the silence belongs to the comment-only block and to nothing else.

`never-multi-line` is the same blindness the other way round. This block runs over three lines and holds a break behind its brace, and nothing is reported:

```css
a {
/*c*/
}
```

Under every one of the three options, and under all four namespaces — `@stylistic/`, `@stylistic/scss/`, `@stylistic/less/` and `@stylistic/styled/`.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/block-opening-brace-newline-after": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json
```

### What did you expect to happen?

`a {/*c*/}` has no newline behind its opening brace, and `always` asks for one there.

### What actually happened?

Nothing is reported, and `--fix` writes nothing.

### Do you have a proposal to fix the bug?

The break the option speaks of has a place to stand — in front of the first comment — so the check has a run to measure and the fix a place to write it. What has to be settled first is whether that run is this rule's to write where the block holds nothing but comments, since the comment is what the check was taught to step over in the first place, and what `never-multi-line` may do with a comment its own fix would have to keep whole.

Found while working on issue #410, which stands on the same walk.
